Dockerfile

```diff
@@ -86,11 +86,11 @@ RUN rm /tmp/requirements.txt
 # TODO: This emits a warning about astroid that is safe to ignore?
 #       https://github.com/PyCQA/astroid/issues/699
```

Comment:
is the todo still relevant?

Reply:
Didn't see any such warning! Will remove.